why-accessibility.qmd

```diff
@@ -0,0 +1,144 @@
+
+Things to check for during development:
+
+-   is the text to complex?
+-   is the language too technical?
+-   are the images described in a way that is compatible with screen
+    readers?
+-   are the buttons and links named in a way that is easy to understand?
+-   are the interactive elements easy to navigate with a keyboard?
+-   are any landmarks or headings used to help with navigation easy to
+    follow?
+-   if we end up making videos or audio content, is there a transcript
+    available?
+
+### Visual accessibility
+
+Visual accessibility is about making sure that the website and the apps
+are easy to use for people with visual impairments. This includes things
+like ensuring that the colour scheme is accessible to people with colour
+blindness, and that the text is easy to read. It also includes things
+like ensuring that the website and the apps are easy to navigate, and
+that the buttons are easy to click on.
+
+Things to check for during development:
+
+-   is the colour scheme accessible to people with colour blindness?
+-   is the colour ration between text and background high enough
+    (minimum 4.5:1)?
+-   is the text easy to read (large enough font)?
+-   is the line length suitable (between 45 and 75 char)?
+-   are the icons easy to decipher?
+-   are the buttons easy to click on (tap-target a minimum of 44x44 px)?
+-   are the interactive states easy to see (default, focus/hover,
+    active/pressed, de-active)?
+
+## Decision outcome
+
+::: content-hidden
+What decision was made, use the form "We decided on CHOICE because of
+REASONS."
+:::
+
+We decided to work on both accessible content and visual accessibility.
+Some of the points highlighted above are things we will need to keep an
+eye out for (like text complexity), others are some that we will be able
+to incorporate into our development process (like ensuring that the
+colour scheme is accessible to people with colour blindness, and that
+buttons have the correct contrast ratio).
+
+There are numerous ways to test for accessibility, and we will need to
+ensure that we have a good testing plan in place. This will at present
+be a semi-manual process, the details of which are described in the
+[Accessibility testing
+plan](!--TODO%20insert%20link%20to%20plan%20once%20written--).
+
+### Consequences
+
+::: content-hidden
+List some potential consequences of this decision.
+:::
+
+One consequence of this decision is that we will need to spend more time
+on development, which will cost more in terms of time and money. Another
+consequence is that we will need to spend more time on testing, which
+will also cost more in terms of time and money.
+
+However, the consequences of not working on accessibility will
+potentially be that our product can't be used by a percentage of our
+potential users. This will limit the potential of the Seedcase project,
+and could potentially lead to a failure of the project. By working on
```

Comment:
Could this be "softened" a bit? I think sounds a bit harsh, maybe?

Reply:
I took a sentence out, did that help?